sacred/exception.py

```diff
@@ -0,0 +1,223 @@
+import inspect
```

Comment:
Having a separate `exception.py` file is a good idea. But I think you forgot to remove the Exceptions from `utils.py` ;-)

Reply:
Oh, I didn't want to push ´exception.py´. At the moment, the exceptions depend on ´utils.py´, and moving the exceptions without an import in ´utils.py´ would be incompatible to the current version.